docs/developer-docs/integrations/bitcoin/submit-transactions.mdx

```diff
@@ -0,0 +1,85 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Sending Bitcoin transactions 
+
+## Overview
+
+To submit transactions to the Bitcoin network, the Bitcoin integration API exposes the following method:
```

Comment:
The canister has to first sign the transaction before sending it to Bitcoin. Signing happens with threshold ECDSA. Wdyt about inlining some of the threshold ECDSA code snippets here as the first step?